templates/table/single_table.html

```diff
@@ -1,2 +1,19 @@
+{% load static %}
 {% load render_table from django_tables2 %}
 {% render_table table "table/tailwind_js_pagination.html" %}
+<script>
+    function initializeRowClickHandlers() {
+        document.querySelectorAll('.clickable').forEach(function(element) {
+            element.addEventListener('click', function() {
+                let editUrl = this.getAttribute('data-redirect-url');
+                window.location.href = editUrl;
```

Comment:
what happens if this is `""`? Might be worth just adding the `clickable` class when there is a url.

Reply:
+1 to not doing anything if the 'editUrl' is empty